express/code/blocks/frictionless-quick-action-mobile/frictionless-quick-action-mobile.css

```diff
@@ -118,6 +118,7 @@
     flex-direction: column;
     font-weight: 900;
     line-height: 1.1;
+    color: var(--color-black);
```

Comment:
This looks Ok. It does seem to obfuscate the issue though. Would it be better to do:

`<meta name="format-detection" content="telephone=no, date=no, address=no, email=no"> `

Reply:
that's the "more complex" solution i was trying to avoid. Having a block injecting a page-level meta for its own looks feels a bit too much, and having it authored also seems unnecessary.